build-tools/src/integTest/groovy/org/elasticsearch/gradle/plugin/PluginBuildPluginFuncTest.groovy

```diff
@@ -75,7 +75,7 @@ class PluginBuildPluginFuncTest extends AbstractGradleFuncTest {
             }
```

Comment:
In general the test that test functionality that moved into the BasePlugin should be moved into a `BasePluginBuildPluginFunc` test

Reply:
but I cannot really test basePluginBuildPlugin without applying `'elasticsearch.esplugin' or `elasticsearch.stable-esplugin`.

Reply:
you can apply the `BasePluginBuildPlugin` just by using the classname

```
        plugins.apply(org.elasticsearch.gradle.plugin. BasePluginBuildPlugin)
```

We do something similar to test _internal_ plugins. see `AbstractGradleInternalPluginFuncTest`